pkg/placement/controller.go

```diff
@@ -76,26 +78,26 @@ const placementDecisionQueueingDelay = 2 * time.Second
 // Controller watches all objects, finds associated placements, when matched a placement wraps and
 // places objects into mailboxes
 type Controller struct {
-	ctx              context.Context
-	logger           logr.Logger
-	ocmClient        client.Client
-	dynamicClient    dynamic.Interface
-	kubernetesClient kubernetes.Interface
-	extClient        apiextensionsclientset.Interface
-	listers          map[string]cache.GenericLister
-	gvkGvrMapper     util.GvkGvrMapper
-	informers        map[string]cache.SharedIndexInformer
-	stoppers         map[string]chan struct{}
-
+	ctx               context.Context
+	logger            logr.Logger
+	ocmClient         client.Client
+	dynamicClient     dynamic.Interface
+	kubernetesClient  kubernetes.Interface
+	extClient         apiextensionsclientset.Interface
+	listers           map[string]cache.GenericLister
+	gvkGvrMapper      util.GvkGvrMapper
+	informers         map[string]cache.SharedIndexInformer
+	stoppers          map[string]chan struct{}
 	placementResolver PlacementResolver
-
-	workqueue     workqueue.RateLimitingInterface
-	initializedTs time.Time
-	wdsName       string
+	workqueue         workqueue.RateLimitingInterface
+	initializedTs     time.Time
+	wdsName           string
+	allowedResources  map[string]bool
```

Comment:
FYI: now that we are no longer stuck on Kubernetes release 1.24 we can use https://github.com/kubernetes/apimachinery/blob/v0.29.1/pkg/util/sets/set.go#L24

Reply:
replaced use of map with sets.Set[string]